app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Resend confirmation instructions" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] [REQUIRED] Great implementation, however, it looks like you skipped a part while implementing the devise. The devise controller part because I can not sign up a new user at the moment nor sign in. You can also take a deep look at the `application controller`.

### Your  Implementation
![Screenshot from 2022-08-24 20-47-29](https://user-images.githubusercontent.com/58692590/186513230-d694a5d4-dcbd-48a9-aaed-40718c758ca9.png)

- [ ] [REQUIRED] Kindly ensure that your design is implemented to match the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) provided and its styles.

### Design Expectation
![Screenshot from 2022-08-24 21-07-31](https://user-images.githubusercontent.com/58692590/186514019-af400276-1039-4cec-a6cb-6f245930f9d4.png)